libs/langchain/tests/integration_tests/vectorstores/test_cratedb.py

```diff
@@ -57,10 +59,18 @@ def prune_tables(engine: sa.Engine) -> None:
     """
     with engine.connect() as conn:
         with Session(conn) as session:
-            from langchain.vectorstores.cratedb import CollectionStore, EmbeddingStore
+            from langchain.vectorstores.cratedb.model import model_factory
 
-            session.query(CollectionStore).delete()
-            session.query(EmbeddingStore).delete()
+            CollectionStore, EmbeddingStore = model_factory(dimensions=1024)
```

Comment:
Is a dummy dimension value here used in order to delete the tables? If so, maybe a comment would be great I think so that users know what to do when facing the same issue (use any dummy value when only used for deletion).

Reply:
Thanks. I've updated the patch and added a corresponding comment.